[2024-04-08T01:10:25.687+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_fetch_pipeline.fetch_data_file_1 manual__2024-04-07T19:20:16+00:00 [queued]>
[2024-04-08T01:10:25.707+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_fetch_pipeline.fetch_data_file_1 manual__2024-04-07T19:20:16+00:00 [queued]>
[2024-04-08T01:10:25.707+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-04-08T01:10:25.729+0530] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_data_file_1> on 2024-04-07 19:20:16+00:00
[2024-04-08T01:10:25.738+0530] {standard_task_runner.py:60} INFO - Started process 47244 to run task
[2024-04-08T01:10:25.747+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'data_fetch_pipeline', 'fetch_data_file_1', 'manual__2024-04-07T19:20:16+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/data_fetch_pipeline.py', '--cfg-path', '/tmp/tmpb5v6ke23']
[2024-04-08T01:10:25.764+0530] {standard_task_runner.py:88} INFO - Job 162: Subtask fetch_data_file_1
[2024-04-08T01:10:25.858+0530] {task_command.py:423} INFO - Running <TaskInstance: data_fetch_pipeline.fetch_data_file_1 manual__2024-04-07T19:20:16+00:00 [running]> on host LAPTOP-HO799B2R.
[2024-04-08T01:10:25.965+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_fetch_pipeline' AIRFLOW_CTX_TASK_ID='fetch_data_file_1' AIRFLOW_CTX_EXECUTION_DATE='2024-04-07T19:20:16+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-07T19:20:16+00:00'
[2024-04-08T01:10:25.967+0530] {data_fetch_pipeline.py:84} ERROR - Error fetching data file 1 for year 2022: '<=' not supported between instances of 'int' and 'str'
[2024-04-08T01:10:25.968+0530] {python.py:201} INFO - Done. Returned value was: None
[2024-04-08T01:10:25.973+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_fetch_pipeline, task_id=fetch_data_file_1, execution_date=20240407T192016, start_date=20240407T194025, end_date=20240407T194025
[2024-04-08T01:10:26.037+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-08T01:10:26.061+0530] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
